### ISSUE TITLE
commands not found

### ISSUE THREAD

**Issue by danielnorkin** [microsoft/vscode-azure-blockchain-ethereum#8]
Issue Type: <b>Bug</b>

Installed extension then try to either connect to consortium and/or create a new service. Even the refresh command not found. Tried to uninstall and reinstall.

I see: 

- command 'azureBlockchainService.connectConsortium' not found
- command 'azureBlockchainService.refresh' not found
- command 'azureBlockchainService.createConsortium' not found
- command 'truffle.newSolidityProject' not found
- etc.

Extension version: 0.1.4
VS Code version: Code 1.35.0 (553cfb2c2205db5f15f3ee8395bbd5cf066d357d, 2019-06-04T01:17:12.481Z)
OS version: Windows_NT x64 10.0.17763

**Comment by caleteeter** [comment]
Sorry to hear this @danielnorkin .  I have seen these errors with bleeding edge version of vscode insiders builds.  Allow me to validate the environment you have to duplicate this to see if there is an issue with the versions of software you have in your environment.

**Comment by danielnorkin** [comment]
> Sorry to hear this @danielnorkin . I have seen these errors with bleeding edge version of vscode insiders builds. Allow me to validate the environment you have to duplicate this to see if there is an issue with the versions of software you have in your environment.

@caleteeter , sure please email me

**Comment by caleteeter** [comment]
Hi again @danielnorkin .  I did some testing and not able to repro the issue.  I tried with older builds of Windows back to 10.0.16.x with no breaks.  Since I last pinged here we have released some updates.  Wondering if on 0.1.5 if you see this same issue.

Additionally, when you start the extension, can provide some info from here:
1) Enable developer tools (under the help menu), see if you are getting any specific errors there.
2) If you bring up the output menu (under the view menu), check the Azure Blockchain output, Logs (Extension Host), and Logs (shared).

Let me know if you see anything in these logs.  Much thanks.
